packages/thirdweb/src/utils/any-evm/zksync/isZkSyncChain.ts

```diff
@@ -20,28 +20,10 @@ export async function isZkSyncChain(chain: Chain) {
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +16]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->would be nice to have it in the chain definition too like `zkSync`  from thirdweb/chains

but i guess if people do defineChain(id) it wont pick that up.

we can iterate on this i guess, fine for now<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4901?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->